zp-relayer/workers/sentTxWorker.ts

```diff
@@ -267,6 +264,10 @@ export async function createSentTxWorker<T extends EstimationType>(gasPrice: Gas
     if (tx.status) {
       return await handleMined(tx, job.data, jobLogger)
     } else {
+      if (txPayload.type === WorkerTxType.DirectDeposit) {
+        const deposits = (txPayload.transactions as DirectDepositTxPayload).deposits
+        await redis.lpush(DIRECT_DEPOSIT_REPROCESS_NAME, ...deposits.map(d => JSON.stringify(d)))
```

Comment:
Can't we use the same `DIRECT_DEPOSIT_SET_NAME` here?

Reply:
For more details:
https://github.com/zkBob/zeropool-relayer/pull/156/#issuecomment-1415799178

Now, there are two different keys: 
1. `DIRECT_DEPOSIT_REPROCESS_NAME` is just a list for re-queued deposits which will be picked up by a special listener in watcher and added to `BatchCache`.
2. `DIRECT_DEPOSIT_SET_NAME` is the primary sorted set which is used to prepare queue batches.

We can't add re-queued deposits directly to `DIRECT_DEPOSIT_SET_NAME` because after each addition a `processCache` function have to be called. I thought that current approach would be the simplest way to implement such communication between relayer and watcher. Wdyt?

Reply:
Now I see, looks ok